packages/d/daisyui.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "daisyui",
+  "description": "Tailwind CSS Components",
+  "filename": "index.js",
+  "homepage": "https://daisyui.com/",
+  "keywords": [
+    "design-system",
+    "tailwindcss",
+    "components",
+    "ui-library",
+    "component",
+    "framework",
+    "tailwind",
+    "theming",
+    "postcss",
+    "design",
+    "css",
+    "ui"
+  ],
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Pouya Saadeghi"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/saadeghi/daisyui.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "daisyui",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*"
+        ]
```

Comment:
We generally like to avoid open globstars as this can lead to us including files that we don't need unexpectedly in future versions. It also looks like this package ships a specific `dist` directory, so my assumption would be that we only need to include files from that and not the whole package?

[Package files](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.%40%28js%7Ccss%29&matches=false&tests=%2F%2F%20Imported%20from%20daisyui%20NPM%20package&tests=LICENSE&tests=dist%2Ffull.css&tests=dist%2Fstyled.css&tests=dist%2Fthemes.css&tests=dist%2Funstyled.css&tests=dist%2Fbase.js&tests=colors%2FcolorNames.js&tests=colors%2Fhex2hsl.js&tests=colors%2Findex.js&tests=index.js&tests=dist%2Fstyled.js&tests=dist%2Fstyled.rtl.js&tests=colors%2Fthemes.js&tests=dist%2Funstyled.js&tests=dist%2Funstyled.rtl.js&tests=dist%2Futilities-styled.js&tests=dist%2Futilities-unstyled.js&tests=dist%2Futilities.js&tests=colors%2Fwindi.js&tests=package.json&tests=CHANGELOG.md&tests=README.md)

Reply:
@MattIPv4 Thanks.  
Files from [`/dist`, `/colors` and also `index.js`](https://unpkg.com/browse/daisyui@1.4.0/) is required so I added another commit to only target those.